apps/web/test-runner.js

```
@@ -0,0 +1,127 @@
+async function findTestFiles() {
+  const files = [];
+  for (const pattern of testPatterns) {
+    const matches = await glob(pattern, { cwd: __dirname });
+    files.push(...matches);
+  }
+  return files.sort();
+}
+
+function runTestBatch(files, batchNumber, totalBatches) {
+  return new Promise((resolve, reject) => {
+    const fileList = files.join(' ');
```

ESLint

[error] 'fileList' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)